test/e2e/examples.go

```diff
@@ -167,6 +167,38 @@ var _ = Describe("Examples e2e", func() {
 			})
 		})
 	})
+
+	Describe("[Skipped][Example]Liveness", func() {
+		It("should create and stop liveness pods", func() {
```

Comment:
nit: Please reword this so that the concatenation of the Describe and It clause forms a fully valid sentence., and accurately describes what the test is testing.  I'd suggest something like:

Describe: "[Skipped][Example]Liveness pods"
It: "should be automatically restarted".